forte/models/ner/model_factory.py

```diff
@@ -151,14 +160,14 @@ def decode(self, input_word, input_char, mask=None, hx=None):
         predicted_tags = [torch.tensor(x).unsqueeze(0) for x in predicted_tags]
         try:
             import texar.torch as texar  # pylint: disable=import-outside-toplevel
-        except ImportError as e:
+        except ImportError as e3:
             raise ImportError(
                 " `texar-pytorch` is not installed correctly."
                 " Consider install texar via `pip install texar-pytorch`."
                 " Or refer to [extra requirement for Texar model support]"
                 "(pip install forte[models])"
                 " for more information."
```

Comment:
This message is better than the previous one, that's why we should actually create a function to centrally manage the messages, instead of copy/pasting them.